packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -715,8 +715,8 @@ class WebViewController {
   /// The Future completes with an error if a JavaScript error occurred.
   ///
   /// When running JavaScript in a [WebView], it is best practice to wait for
-  //  the [WebView.onPageFinished] callback. This guarantees all the JavaScript
-  //  embedded in the main frame HTML has been loaded.
+  ///  the [WebView.onPageFinished] callback. This guarantees all the JavaScript
+  ///  embedded in the main frame HTML has been loaded.
```

Comment:
nit: the spacing is slightly off

```suggestion
  /// the [WebView.onPageFinished] callback. This guarantees all the JavaScript
  /// embedded in the main frame HTML has been loaded.
```